gravity.sh

```diff
@@ -269,6 +269,10 @@ gravity_DownloadBlocklistFromUrl() {
     else port=80
     fi
     bad_list=$(pihole -q -adlist "${domain}" | head -n1 | awk -F 'Match found in ' '{print $2}')
+    # added check for https://github.com/pi-hole/pi-hole/issues/3196
+    if [ $(echo -n ${bad_list} | wc -m) -eq 0 ]; then
```

Comment:
No need to `echo` and `wc`, just check the `bad_list` variable directly for content. 
```suggestion
    if [-z "${bad_list}" ]; then
```